l10n/de/viewer.properties

```diff
@@ -82,13 +85,19 @@ document_properties_version=PDF-Version:
 document_properties_page_count=Seitenzahl:
 document_properties_close=Schließen
 
+print_progress_message=Dokument wird zum Drucken vorbereitet…
+# LOCALIZATION NOTE (print_progress_percent): "{{progress}}" will be replaced by
+# a numerical per cent value.
+print_progress_percent={{progress}}%
+print_progress_close=Abbrechen
+
 # Tooltips and alt text for side panel toolbar buttons
 # (the _label strings are alt text for the buttons, the .title strings are
 # tooltips)
 toggle_sidebar.title=Sidebar umschalten
 toggle_sidebar_label=Sidebar umschalten
-outline.title=Dokumentstruktur anzeigen
-outline_label=Dokumentstruktur
+document_outline.title=Dokumentstruktur anzeigen
```

Comment:
Nit; Note that the reason for changing the l10n id, was that additional information was added, see e.g. https://github.com/mozilla/pdf.js/blob/7d8fa1385d47e180cdba32a2e63d87d5c4132d7f/l10n/en-US/viewer.properties#L105.